GPU/TPCSpaceChargeBase/AliTPC3DCylindricalInterpolatorIrregular.cxx

```diff
@@ -622,6 +623,10 @@ void AliTPC3DCylindricalInterpolatorIrregular::SetValue(
 {
   Int_t indexInner;
   Int_t index;
+  TMatrixD* mat;
```

Comment:
I don't understand this change? The scope of the variables can remain in the for loop, as suggested by Codacy. Or do I miss something?